<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Exporter</class>
 <widget class="QWidget" name="Exporter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>473</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>578</width>
    <height>473</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>A3Plus Exporter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>3</number>
       </property>
       <item>
        <widget class="QGroupBox" name="corelGroup">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>81</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>81</height>
          </size>
         </property>
         <property name="title">
          <string>Corel Draw</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="verticalSpacing">
           <number>1</number>
          </property>
          <item row="1" column="0">
           <widget class="QCheckBox" name="kurvaOto">
            <property name="toolTip">
             <string>Kurvakan font sebelum export</string>
            </property>
            <property name="text">
             <string>Kurva Otomatis</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Gunakan Versi</string>
            </property>
            <property name="buddy">
             <cstring>comboVersi</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboVersi">
            <property name="toolTip">
             <string>Pilihan Versi CorelDRAW</string>
            </property>
            <property name="styleSheet">
             <string notr="true">text-align:&quot;right&quot;</string>
            </property>
            <property name="editable">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>Auto</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QToolButton" name="btPdfSetting">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Buka PDF Setting</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="fileGroup">
         <property name="minimumSize">
          <size>
           <width>251</width>
           <height>79</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>251</width>
           <height>79</height>
          </size>
         </property>
         <property name="title">
          <string>File</string>
         </property>
         <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,1">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <property name="horizontalSpacing">
           <number>4</number>
          </property>
          <property name="verticalSpacing">
           <number>6</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Terbuka :</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>leDoc</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="leDoc">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QToolButton" name="tbDet">
            <property name="toolTip">
             <string>Deteksi otomatis file terbuka</string>
            </property>
            <property name="text">
             <string>Deteksi</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Export ke :</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>leExpF</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leExpF">
            <property name="toolTip">
             <string>Lokasi Export</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QToolButton" name="tbBrowse">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Pilih export folder</string>
            </property>
            <property name="text">
             <string>Pilih</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="paramGroup">
         <property name="minimumSize">
          <size>
           <width>251</width>
           <height>251</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>251</width>
           <height>251</height>
          </size>
         </property>
         <property name="title">
          <string>Parameter</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="horizontalSpacing">
           <number>6</number>
          </property>
          <item row="5" column="4">
           <widget class="QPushButton" name="pbExport">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Export</string>
            </property>
            <property name="shortcut">
             <string>Ctrl+E</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>File</string>
            </property>
            <property name="buddy">
             <cstring>leFile</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="4">
           <widget class="QLineEdit" name="leBahan">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Nama bahan</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="1" colspan="4">
           <widget class="QPlainTextEdit" name="txKet">
            <property name="toolTip">
             <string>Keterangan tambahan</string>
            </property>
            <property name="tabChangesFocus">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Klien</string>
            </property>
            <property name="buddy">
             <cstring>leKlien</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Keterangan</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="buddy">
             <cstring>txKet</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QCheckBox" name="sideCheck">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Keterangan double side (BB)</string>
            </property>
            <property name="text">
             <string>BB</string>
            </property>
           </widget>
          </item>
          <item row="3" column="4">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Qty</string>
              </property>
              <property name="buddy">
               <cstring>leQty</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leQty">
              <property name="toolTip">
               <string>Keterangan jumlah halaman dan duplikasi</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Bahan</string>
            </property>
            <property name="buddy">
             <cstring>leBahan</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Halaman</string>
            </property>
            <property name="buddy">
             <cstring>lePage</cstring>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="4">
           <widget class="QProgressBar" name="pBar">
            <property name="value">
             <number>0</number>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="format">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lePage">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>55</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Halaman yang akan diexport</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="4">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QComboBox" name="comboBox">
              <property name="minimumSize">
               <size>
                <width>91</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>91</width>
                <height>22</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Diexport oleh</string>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leFile">
              <property name="toolTip">
               <string>Nama file export</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1" colspan="4">
           <widget class="QLineEdit" name="leKlien">
            <property name="toolTip">
             <string>Nama Konsumen</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="histGroup">
       <property name="minimumSize">
        <size>
         <width>268</width>
         <height>411</height>
        </size>
       </property>
       <property name="title">
        <string>Riwayat</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="verticalSpacing">
           <number>11</number>
          </property>
          <item row="1" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Filter</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>leFilter</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QLineEdit" name="leFilter">
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="pbFilter">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Ok</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QComboBox" name="cbxViewRange">
            <item>
             <property name="text">
              <string>Hari ini</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Semua</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Lihat</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>cbxViewRange</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="histTable">
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>50</number>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>20</number>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>20</number>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="btFirst">
            <property name="text">
             <string>&lt;&lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btPrev">
            <property name="text">
             <string>&lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spHalaman">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="suffix">
             <string>/1</string>
            </property>
            <property name="prefix">
             <string>Page </string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btNext">
            <property name="text">
             <string>&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btLast">
            <property name="text">
             <string>&gt;&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="pbDetach">
            <property name="text">
             <string>Layar Penuh</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="status" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>432</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Tentang ?</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionToggleHistory">
   <property name="text">
    <string>Toggle History</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionRefreshHistory">
   <property name="text">
    <string>Refresh History</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionCloseDocument">
   <property name="text">
    <string>Close ActiveDocument</string>
   </property>
   <property name="toolTip">
    <string>Close Current ActiveDocument</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tbDet</tabstop>
  <tabstop>leExpF</tabstop>
  <tabstop>tbBrowse</tabstop>
  <tabstop>leKlien</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>leFile</tabstop>
  <tabstop>leBahan</tabstop>
  <tabstop>lePage</tabstop>
  <tabstop>sideCheck</tabstop>
  <tabstop>leQty</tabstop>
  <tabstop>txKet</tabstop>
  <tabstop>pbExport</tabstop>
  <tabstop>leFilter</tabstop>
  <tabstop>cbxViewRange</tabstop>
  <tabstop>pbFilter</tabstop>
  <tabstop>histTable</tabstop>
  <tabstop>btPdfSetting</tabstop>
  <tabstop>btFirst</tabstop>
  <tabstop>comboVersi</tabstop>
  <tabstop>btPrev</tabstop>
  <tabstop>spHalaman</tabstop>
  <tabstop>btNext</tabstop>
  <tabstop>btLast</tabstop>
  <tabstop>pbDetach</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>kurvaOto</tabstop>
  <tabstop>leDoc</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>leFilter</sender>
   <signal>returnPressed()</signal>
   <receiver>pbFilter</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
